src/renderer/components/uielements/button/button.tsx

```diff
@@ -41,6 +42,7 @@ const Button: React.FC<Props> = (props: Props): JSX.Element => {
       sizevalue={sizevalue}
       typevalue={typevalue}
       round={round}
+      loading={loading}
```

Comment:
No default value is needed for `loading` - it's already provided by `antd` button